packages/models/newApplicant.ts

```diff
@@ -11,12 +11,12 @@ export interface NewApplicant {
   commitment: number;
   extraInfo?: string;
   portfolioLink?: string;
+  yearsExperience?: number;
```

Comment:
this field is nowhere to be found in the api response.

@becevka  do we plan to add this to applicants collection?